packages/atomic/src/components/facets/atomic-category-facet/atomic-category-facet.tsx

```diff
@@ -156,6 +160,15 @@ export class AtomicCategoryFacet
   @Prop() public injectionDepth = 1000;
   // @Prop() public customSort?: string; TODO: KIT-753 Add customSort option for facet
 
+  @FocusTarget()
+  protected showMoreFocus!: FocusTargetController;
+
+  @FocusTarget()
+  protected headerFocus!: FocusTargetController;
+
+  @FocusTarget()
+  protected activeValueFocus!: FocusTargetController;
```

Comment:
The `activeValueFocus` here is a more complex use case that the above two. Depending on the state after the search, we don't necessarily want to focus the same element. I recommend taking a look at where both of its methods are called.